app/src/routes/v3/forest-watcher.router.ts

```diff
@@ -7,6 +7,10 @@ const TemplatesService = require("services/template.service");
 const AreaValidator = require("validators/area.validator");
 const moment = require("moment");
 const config = require("config");
+const AreaTemplateRelationService = require("services/areaTemplateRelationService").default;
+
+import mongoose from "mongoose";
+import { AreaTemplateRelationModel } from "../../models";
```

Comment:
You can use absolute paths for the imports if you like.

`import { AreaTemplateRelationModel } from "models";`